src/baseballGame/baseballGame.js

```diff
@@ -0,0 +1,58 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { GAME_MSG, ERROR_MSG } from "../Messages";
+import { getComputerNum, getUserNum } from "./getValue";
+
+// 3. 컴퓨터와 유저의 숫자 비교
+export async function compareNum() {
+  const computer_num = getComputerNum();
```

Comment:
JS에서 권장하는 변수 선언 방식은 lowerCamelCase입니다!!
```suggestion
  const computerNumber = getComputerNum();
```

Reply:
앗 감사합니다! 대문자로 작성했다가 소문자로 변경하면서 이런 실수를 ㅠㅠ 종종 헷갈렸는데 이제는 머리에 박혀서 안잊어버릴 것 같아요!